python/tests/test_variable.py

```diff
@@ -1010,3 +1010,27 @@ def test_equal():
 def test_not_equal():
     var = sc.Variable()
     assert_export(sc.not_equal, x=var, y=var)
+
+
+def test_radd_int():
+    var = sc.Variable(dims=['x'], values=[1, 2, 3])
+    assert (var + 1).dtype == sc.dtype.int64
+    assert (1 + var).dtype == sc.dtype.int64
```

Comment:
So on Windows `var` will be `int32` I believe, and the addition promotes to `int64`? I think we may have this issue in other, existing overloads as well, so not necessarily something that needs to be fixed here.

Reply:
Beyond duplicating the exports (for both `int32` and `int64`) I'm not sure of an alternative.

Reply:
I suppose that is what we might need to do, but I suggest we can leave that for another time.